api/controllers/userCtrl.js

```diff
@@ -606,6 +606,23 @@ module.exports.getAllExercises = function(req, res) {
 }
 
 
+module.exports.updateMaxs = function(req, res) {
```

Comment:
change this to updateMaxes

Reply:
I changed this in another PR, sorry I didn't see this comment